EventView-1/src/main/java/com/eventview/model/EventsPayload.java

```diff
@@ -39,9 +40,9 @@ public void setEventDate(String eventDate) {
 		this.eventDate = eventDate;
 	}
 
-	public EventsPayload(Integer eventId, String fullName, String eventType, String eventDate) {
+	public EventsPayload(Integer eventId, String fName, String lName, String eventType, String eventDate) {
 		this.eventId = eventId;
-		this.fullName = fullName;
+		this.fullName = fName + " " + lName;
```

Comment:
use apache StringUtil library here , read abt it. avoid + to concat any strings in future also